jest.config.js

```diff
@@ -0,0 +1,23 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+const path = require('path');
+
+module.exports = {
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: "coverage",
+
+  // The test environment that will be used for testing
+  testEnvironment: "node",
+
+  // The glob patterns Jest uses to detect test files
+  testMatch: [
+  //   "**/__tests__/**/*.[jt]s?(x)",
```

Comment:
Could we / should we delete this comment?

Reply:
Can do. Fixed